models/issues/review.go

```diff
@@ -243,6 +243,7 @@ type FindReviewOptions struct {
 	IssueID      int64
 	ReviewerID   int64
 	OfficialOnly bool
+	LatestOnly   bool // Latest per-reviewer
```

Comment:
After reading the code again, I think it's better to add a `FindLatestReviews` , instead of adding `LatestOnly` to FindReviewOptions.

The reason is, the newly added `LatestOnly` is not respected by `toCond`, but `FindReviewOptions` is shared by `FindReviewOptions` and `CountReviewOptions`. So, if LatestOnly=true, FindReviewOptions and CountReviewOptions behave differently, it would cause bugs or confuse developers in the future.

----

If we introduce a `FindLatestReviews`, then everything is clear.

Reply:
That's a very good point, thanks for the feedback. I've made the requested revisions.